### DISCUSSION TITLE
How to add added filed on existing type to customProductMappings to index in elastic search

### DISCUSSION THREAD

**Discussion by ramkishore-yerubandi** [vendurehq/vendure#1653] [Q&A]
I have added a type to ProductVariant Resolver and need this information in the search results of a product variant. Tried using customeProductMappings configuration on ElasticSearch Plugin. As this type is not available on ProductVariant Entity I could not achieve the desired output. Is there any way where I can add computed filed on Entity? or any alternative to achieve this?

Kindly help.

**Comment by michaelbromley** [comment]
Hi,

Currently in a custom mapping there is no way to run arbitrary async code to fetch other data. Only the actual relations of the Product or ProductVariant can be joined.

So an alternative would be to actually store this data in a customField of the ProductVariant perhaps?
